eox_tenant/templatetags/ednx.py

```diff
@@ -34,14 +34,14 @@ def microsite_css_overrides_file():
     Django template tag that outputs the css import for a:
     {% microsite_css_overrides_file %}
     """
-    file_path = microsite.get_value('css_overrides_file', None)
+    file_path = configuration_helpers.get_value('css_overrides_file', None)
```

Comment:
I think this line is not necessary. @jfavellar90  What do you think?

Reply:
@morenol agree with it, let's procees to merge this one